### PR TITLE
Add condition to app service certificate deployment

### PR DESCRIPTION
**Pull request by OutKa5t**
### Context
UCAS env release broken. This is the fix. 

### Changes proposed in this pull request
Adding handle on custom domain name to stop it borking where no custom host name is being used.
 
### Guidance to review
erm...

### CODE REVIEW

File: azure/template.json [hunk 1]
```diff
@@ -1,261 +1,262 @@
...
-                "parameters": {
-                    "appServicePlanName": {
-                        "value": "[variables('appServicePlanName')]"
-                    }
-                }
-            }
+    {
+      "name": "app-service-certificate",
+      "type": "Microsoft.Resources/deployments",
+      "condition": "[greater(length(parameters('certificateName')), 0)]",
```
Comment (OutKa5t) [inline azure/template.json]: @dankmitchell This is the change.

Comment (tvararu) [reply azure/template.json]: It's easy to see with whitespace changes disabled. :)

Comment (dankmitchell) [reply azure/template.json]: No idea what that does but ship it :)

**Comment by EwanNoble** [comment]
Deployment succeeded here: https://dev.azure.com/dfe-ssp/Become-A-Teacher/_releaseProgress?releaseId=651&_a=release-environment-logs&environmentId=1324

File: azure/template.json (hunk 1 above)
Comment (OutKa5t) [reply azure/template.json]: You know what @tvararu I never knew that toggle was in github. I just thought it was shit. Thanks for the pointer.

Comment (dankmitchell) [reply azure/template.json]: Come and spend some more time down south and we can learn from each other ;-)

**Pull request merged**